QuizApp/QuizListCollectionViewCell.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+import UIKit
+import SnapKit
+import Kingfisher
+
+class QuizListCollectionViewCell: UICollectionViewCell {
+
+    static let id = "QuizListCollectionViewCell"
+
+    private var quiz: QuizModel!
+    private var quizImageView: UIImageView!
```

Comment:
```suggestion
    private var imageView: UIImageView!
```

in this case it's redundant to prefix views names with quiz, so remove it to shorten the names.